.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           name: Running tests
           command: |
             . .venv/bin/activate
-            cp .env.template .env.test
+            cp .env.citest .env.test
```

Comment:
I suggest we rename `.env.citest` to `.env.test` and add it to git 
Could it be an issue ? We could remove this `copy` step

Reply:
.env.test is already used by locat test. This PR remove the gitignore of this file